content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,390 @@
+This parameter is only useful in the context of the Memory Manager feature. 
+The Memory Manager will not use this reserved memory for the allocation of container workloads.
+
+For example, if you have a NUMA0 with `10Gi` of memory available, and the `--reserved-memory` was specified to reserve `1Gi` of memory at NUMA0, the Memory Manager assumes that only `9Gi` is available for containers.
+
+You can omit this parameter, however, you should be aware that the amount of reserved memory from all NUMA nodes should be equal to the amount of memory specified by the [Node Allocatable features]((https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/)). If at least one node allocatable parameter is non-zero, you will need to specify `--reserved-memory` for at least one NUMA node. In fact, `eviction-hard` threshold value is equal to `100Mi` by default, so if `static` policy is used, `--reserved-memory` is obligatory.
+
+Also, avoid the following configurations:
+1. duplicates, i.e. the same NUMA node or memory type, but with a different value;
+2. setting zero limit for any of memory types;
+3. NUMA node IDs that do not exist in the machine hardware;
+4. memory type names different than `memory` or `hugepages-<size>` (hugepages of particular `<size>` should also exist).
+
+Syntax: 
+
+`--reserved-memory N:memory-type1=value1,memory-type2=value2,...`
+* `N` (integer) - NUMA node index, e.g. `0`
+* `memory-type` (string) - represents memory type:
+  * `memory` - conventional memory
+  * `hugepages-2Mi` or `hugepages-1Gi` - hugepages 
+* `value` (string) - the amount of reserved memory, e.g. `1Gi`
+
+Example usage:
+
+`--reserved-memory 0:memory=1Gi,hugepages-1Gi=2Gi`
+
+or
+
+`--reserved-memory 0:memory=1Gi --reserved-memory 1:memory=2Gi`
+
+When you specify values for `--reserved-memory` flag, you must comply with the setting that you prior provided via Node Allocatable Feature flags. That is, the following rule must be obeyed for each memory type: 
+
+`sum(reserved-memory(i)) = kube-reserved + system-reserved + eviction-threshold`, 
+
+where `i` is an index of a NUMA node. 
+
+If you do not follow the formula above, the Memory Manager will show an error on startup.
+
+In other words, the example above illustrates that for the conventional memory (`type=memory`), we reserve `3Gi` in total, i.e.: 
+
+`sum(reserved-memory(i)) = reserved-memory(0) + reserved-memory(1) = 1Gi + 2Gi = 3Gi`
+
+An example of Node Allocatable Feature flags [configuration](https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/):
+* `--kube-reserved=cpu=500m,memory=50Mi`
+* `--system-reserved=cpu=123m,memory=333Mi`
+* `--eviction-hard=memory.available<500Mi`
+
+_NOTICE: hard eviction threshold is not equal to zero by default but `100Mi`, so do not forget to decrease the total amount set via `--reserved-memory` by this `100Mi`. Otherwise, the Memory Manager will display an error. Here is an example of a correct configuration:_

```

Comment:
You can use a callout shortcode. Additionally, let's explain the behavior if the memory numbers are incorrect.
```suggestion
{{< note >}}
The default hard eviction threshold is 100MiB, and **not** zero. Remember to decrease the amount of memory that you reserve by setting `--reserved-memory` by that hard eviction threshold. Otherwise, the kubelet [insert details of what the kubelet does in this case].

  Here is an example of a correct configuration:
```shell
--feature-gates=MemoryManager=true 
--kube-reserved=cpu=4,memory=4Gi 
--system-reserved=cpu=1,memory=1Gi 
--memory-manager-policy=static 
--reserved-memory 0:memory=3Gi --reserved-memory 1:memory=2148Mi
```
{{< /note >}}

Reply:
For shortcode details see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/ and https://kubernetes.io/docs/contribute/style/style-guide/